app/components/fruit-tip.hbs

```diff
@@ -0,0 +1,27 @@
+{{#if this.showingFruitChoices}}
+  <div class="flex bg-df-pink border-dashed border-white border-4 p-4 absolute p-3 -mt-40 ml-4">
+    <button
+      {{on "click" this.hideFruitChoices}}
+      class="cool-button"
+      type="button"
+      >
+      X
+    </button>
+    {{#each this.fruitTypes as |fruit|}}
+      <button
+        {{on "click" (fn this.fruitTip fruit.name)}}
+        class="cool-button fruit-tip-button"
+        type="button"
+        >
+        <img src={{fruit.image}} />
+      </button>
+    {{/each}}
+  </div>
+{{/if}}
+<button
+  {{on "click" this.showFruitChoices}}
+  class="cool-button fruit-tip-button"
+  type="button"
+  >
+  give the DJ fruit
```

Comment:
was thinking this needed to be in the translations, but what would be the best way of phrasing this across all of the locales? Maybe the button text could be a fruit emoji and the title is "give the DJ fruit".

Reply:
Yes I think emoji with the text in the mouse hover title would be much better :strawberry: